tests/base_tests.py

```diff
@@ -65,6 +65,13 @@ def __init__(self, *args, **kwargs):
                 appbuilder.sm.find_role('Gamma'),
                 password='general')
 
+        gamma2 = appbuilder.sm.find_user('gamma2')
```

Comment:
could u create function `find_or_create(user, user_group)` - looks like we have a lot of repetitive code here.
[optional]